src/internet_identity/src/assets.rs

```diff
@@ -26,7 +26,10 @@ lazy_static! {
     // The <script> tag that sets the canister ID and loads the 'index.js'
     static ref INDEX_HTML_SETUP_JS: String = {
         let canister_id = api::id();
-        format!(r#"var canisterId = '{canister_id}';let s = document.createElement('script');s.async = true;s.src = 'index.js';document.head.appendChild(s);"#)
+
+        let version_info = crate::arg_data().version_info.unwrap_or("no version info".to_string());
```

Comment:
Thought about it, and thought it'd also be good in the future if we have automated mainnet tests that poll for a new version to be deployed; although it's a bit more finicky. You call!

Reply:
Actually, have the extra canister call is really a waste of resources I feel. What makes you uneasy about this?

Reply:
> it'd also be good in the future if we have automated mainnet tests that poll for a new version to be deployed

Yes, this would be great.

> Your call!
If you have the time to implement it using a canister call, I would appreciate that. But if not, it's better to merge this than not having version information at all.

Reply:
Ah, that's a good solution too 👍